code/stack.yaml

```diff
@@ -58,6 +58,11 @@ extra-deps:
 - unicode-names-3.2.0.0@sha256:fef7241d93170e26265e84553090253a5e8ee207645d47cf271816f5834d07d2
 - unicode-properties-3.2.0.0@sha256:239766a6ac4322329353f6b9cd546024fd8c5f0235c8e32b3cba2d6bd245a699
 
+# Used only for a basic set of flags for haddock
+build:
+  haddock-deps: false
+  haddock-hyperlink-source: false
```

Comment:
Why?

Reply:
These were the original flags used for generating the Haddock docs. `haddock-deps` is a switch for building haddocks for dependencies, and `haddock-hyperlink-source` is for creating the links to the source code. I think we can re-enable that, we'd just need to tweak the settings a little bit for it to go to github.

Reply:
Right, I was only asking about the hyperlinks. It would be nice to have that.

Reply:
I'm sure we can figure it out. I'll make a ticket.